index.js

```diff
@@ -47,6 +48,16 @@ class Client {
         })
     })
   }
+
+  listAnalyses (queryString) {
+    let options = {
```

Comment:
`options` should be a `const`.

Reply:
This one I do so confidently that I just made the change. In general, I am trying to remove the back and forth on things to speed things up. If I think I can just get it done right, I'd rather not bother you.